docs/moduleAna3AIMEC.rst

```diff
@@ -25,6 +25,9 @@ Inputs
 *  Results from avalanche simulation (when using results from com1DFA,
    the helper function :py:func:`ana3AIMEC.dfa2Aimec.mainDfa2Aimec` in :py:mod:`ana3AIMEC.dfa2Aimec` fetches and prepares the input for Aimec)
 
+.. Note:: The DEM file can have a different spatial resolution compared to the result files. However, the z coordinates
```

Comment:
Add a quick note as to the reason for this, i.e. something like 'due to remeshing in eg. com1DFA' or so...